### PR TITLE
[Remove Stream/Optional usage return value part:analytic]Lps 171993

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-171993

### CODE REVIEW

**Comment by slnn** [comment]
Forced pushed with SF changes, no logic change, rerun tests since the previous tests do not finished

**Comment by slnn** [comment]
ci:test:sf

**Comment by slnn** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
@slnn This pull https://github.com/liferay-core-infra/liferay-portal/pull/2892 has some changes that you also made here (the interface in info-api and an impl in analytics). Could you see if you can exclude the change from your pull? Basically I'd prefer to include such change in the pull that actually aims to modify the api module.

File: modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java [hunk 1]
```diff
@@ -55,24 +54,36 @@ public List<Action> getActions() {
...
-		).orElse(
-			StringPool.BLANK
-		);
+		User user = _getUser(journalArticle);
+
+		if (user == null) {
+			return StringPool.BLANK;
+		}
+
+		String fullName = user.getFullName();
```
Comment (dantewang) [inline modules/dxp/apps/analytics/analytics-reports-journal-impl/src/main/java/com/liferay/analytics/reports/journal/internal/info/item/JournalArticleAnalyticsReportsInfoItem.java:63]: @slnn Could you double check if `user.getFullName()` can return null?

**Comment by slnn** [comment]
OK, I will resend this one after https://github.com/liferay-core-infra/liferay-portal/pull/2892 merged

**Comment by dantewang** [comment]
@slnn Does this change affect other changes in your pull? If not, you can go ahead and send the other changes.

**Comment by slnn** [comment]
I have split this pull with different part but they are only contains method return value, so I need to revisit this module to remove other Stream/Optional usages, I am working on other modules now. So I would like to revisit this one later.

**Pull request closed without merging**